index.html

```diff
@@ -0,0 +1,177 @@
+                    <div class="icon"><i class="fas fa-handshake"></i></div>
+                    <div class="image">Unity</div>
+                    <div class="text">Listen to the speakers from various countries about the messages of sharing and opening.</div>
+                </div>
+            </div>
+
+            <div class="items">
+                <div class="cards">
+                    <div class="icon"><i class="fas fa-people-carry"></i></div>
+                    <div class="image">
+                    
+                        Agriculture</div>
+                    <div class="text">Listen to the speakers from various countries about the messages of sharing and opening.</div>
+                </div>
+            </div>
+        </div>
+        <div class="main_button">Join Visit Rwanda</div>
+        <div class="main_button2"><a href="">SEE THE WHOLE PROGRAM</a> </div>
+    </main>
+
+    <div class="content" id="contents">
+        <h2 class="title_content">Featured speakers</h2>
+        <div class="hr"></div>
+        <div class="content_container">
+        <div class="content_item" id="item1">
+            <div class="image_content">
+                <img src="images/kagame.jpg" alt="">
+            </div>
+             <div class="text_content">
+                 <h2>Rwanda</h2>
+                 <h4>His President of the Republic of Rwanda. </h4>
+                 <h5>
+                    He served as Chair of the AU from 2018 to 2019 and chaired the East African Community from 2018-2021.
+                 </h5>
+             </div>
+            </div>
+        </div>
+    </div>
+    <div class="more_btn">
+        <button>More <i class="fas fa-angle-double-down"></i></button>
+    </div>
+    <div class="less_btn itemCard">
+        <button>Less <i class="fas fa-angle-double-up"></i></button>
+    </div>
+    <footer class="foot">
+        <div class="partner" id="sponsor">
+            <h2>Partner</h2>
+        <div class="hr"></div>
+            <div id="sponsors">
+            <div id="sponsor_1">mozilla</div>
+            <div id="sponsor_2">Google</div>
+            <div id="sponsor_3">NAVER</div>
+            <div id="sponsor_4"></div>
+            <div id="sponsor_5">daumkakao</div>
+            <div id="sponsor_6">NAVER</div>
+        </div></div>
+        <div class="address">
+            <div class="creative"><img src="images/gorilla-logo.png" alt="image">
+          </div>
+          <div class="logoname">Gorila Creative</div>
+            <div class="footer_text">
+                <h3>2021 Creative Commons Rwanda. Some Rights Reserved.</h3>
+                <p>
+                    His Excellency Paul Kagame, President of Rwanda, and Commonwealth Secretary-General, The Rt. <br> Hon. Patricia Scotland QC, are today announcing the postponement of CHOGM 2021 as a result of <br> the continuing impact of the Covid-19 pandemic.  
+                </p>
+
+            </div>
+        </div>
+    </footer>
```

Comment:
The **background color** of your footer on both the mobile and desktop versions are the same. As specified let's ensure to improve upon these aspects:
- [x]  The footer, on mobile size should have a white background, which is implemented.
- [ ] The footer, on desktop size should have a dark background.